lib/moon/components/menu_item/radio.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Moon.Components.MenuItem.Radio do
+  @moduledoc false
+
+  use Moon.StatelessComponent
+
+  prop(is_selected, :boolean, from_context: {Moon.Components.MenuItem, :is_selected})
+
+  def render(assigns) do
+    ~F"""
+    <span class="flex w-6 h-6 justify-center items-center">
+      <span class={
+        "block relative w-4 h-4 rounded-full shadow-[0_0_0_1px_inset] ",
+        (@is_selected && "shadow-piccolo") || "shadow-trunks"
+      }>
+        <span class={
+          (@is_selected && "scale-100") || "scale-0",
```

Comment:
This should be Radio component.

Reply:
i'd like to divide look & form logic inside the original Components.RadioButton component and reuse it's look (very few code) in the MenuItem.Radio